doc/content/getting-started/ttn/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "The Things Network"
+description: ""
+distributions: "The Things Network"
+---
+
+{{< figure src="logo_TTN.svg" class="float plain" width="200px" >}}
+
+The Things Network is a global community building an open, crowdsourced, decentralized LoRaWAN network.
+
+The Things Network runs {{% tts %}} Community, and is a great way to get started testing devices, applications, and integrations, and get familiar with LoRaWAN.
+
+<!--more-->
+
+[Login or create an account](https://console.cloud.thethings.network/) to get started with The Things Network and start using {{% tts %}} Console.
+
+Once you have an account, get started by following instructions for adding [Devices]({{< ref "/devices" >}}), [Gateways]({{< ref "/gateways" >}}), and [Integrations]({{< ref "/integrations" >}}).
+
+For more information about The Things Network and LoRaWAN, see [The Things Network Documentation](https://thethingsnetwork.org/docs)
+
+For commercial solutions, we recommend using an SLA-backed network provided by The Things Industries. Learn more about [The Things Industries](https://thethingsindustries.com/).
+
+## Clusters
+
+The Things Network is a multi-cluster deployment. This means that while your account information is stored in a central location, you can connect your gateways to a closer cluster, and route all your IoT traffic in that cluster. This can significantly reduce latency because your traffic would not have to cross half the planet.
```

Comment:
```suggestion
{{% tts %}} Community Edition by The Things Network is a multi-cluster deployment. We use these clusters for different geographic regions, which means that while your account information is stored in a central location, you can connect your gateways to a closer cluster, and route all your IoT traffic in that cluster. This can significantly reduce latency because your traffic would not have to cross half the planet.
```